hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FileSystemConfigurations.java

```diff
@@ -119,6 +119,9 @@ public final class FileSystemConfigurations {
   public static final int STREAM_ID_LEN = 12;
   public static final boolean DEFAULT_ENABLE_ABFS_LIST_ITERATOR = true;
 
+  public static final boolean DEFAULT_FS_AZURE_REDIRECT_RENAME = false;
+  public static final boolean DEFAULT_FS_AZURE_REDIRECT_DELETE = true;
+
```

Comment:
lets add a comment on why a one config is true and why other is false.

Reply:
Done